auditeval/test.go

```diff
@@ -117,14 +136,20 @@ type Tests struct {
 	BinOp     binOp       `yaml:"bin_op"`
 }
 
-func (ts *Tests) Execute(s string) (result bool) {
-	if ts == nil || s == "" {
-		return result
+func (ts *Tests) Execute(s string) *testOutput {
```

Comment:
See comment below, I'm not sure we gain much by returning a testOutput structure from Execute as we immediately want to split the bool and the string out again

Reply:
IMO, in this case using a structure prevents code redundancy and possible changes of the function declarations in all the places in case another value is added. Also it allows for the unit-tests not to be affected, as the test will simply not refer to the value in the struct it does not need. Thus for this case, I think that we gain flexibility. 

It is true, that golang supports mutliple-valued functions, but I don't think this is the right case for use. Of course we can change the code if this is necessary.

Reply:
We may have to agree to disagree on this one - I'd say it's more idiomatic Go to split them out (because it is both easier to read and more succinct) but I agree that it is not a functional difference (though the nil check is then required above if we go this way).

Reply:
Of course, handling nil check is in progress.